bin/release_android

```diff
@@ -17,22 +17,41 @@ if [ -z "$VERSION" ]; then
     exit 1
 fi
 
-USER=$2
-if [ -z "$USER" ]; then
-    print_error "Missing Bintray API user"
-    exit 1
-fi
+URL="https://oss.sonatype.org/service/local/staging/deploy/maven2"
 
-TOKEN=$3
-if [ -z "$TOKEN" ]; then
-    print_error "Missing Bintray API token"
-    exit 1
+if [ "$VERSION" == "0.0.0-dev" ]; then
+    VERSION="0.0.0-SNAPSHOT"
+    URL=https://oss.sonatype.org/content/repositories/snapshots
 fi
 
-sed -i -E "s|<version>.+</version>|<version>$VERSION</version>|g" bin/package/android/pom.xml
+sed -i -E "s|<version>.+</version>|<version>$VERSION</version>|g" bin/package/android/mvn.pom
 
-URL=https://api.bintray.com/content/mysterium/Node/mobile-node/$VERSION/mysterium/network/mobile-node/$VERSION
+mvn gpg:sign-and-deploy-file -e -X \
+    -settings=bin/package/android/mvn.settings \
+    -Durl=$URL \
+    -DrepositoryId=ossrh \
+    -Dgpg.keyname=336826E2222A0B29 \
+    -DpomFile=bin/package/android/mvn.pom \
+    -Dfile=build/package/Mysterium.aar
+
+if [ "$VERSION" == "0.0.0-SNAPSHOT" ]; then
+    exit 0
+fi
 
-curl -T bin/package/android/pom.xml         -u$USER:$TOKEN "$URL/mobile-node-$VERSION.pom?override=1&publish=1"
-curl -T build/package/Mysterium.aar         -u$USER:$TOKEN "$URL/mobile-node-$VERSION.aar?override=1&publish=1"
-curl -T build/package/Mysterium-sources.jar -u$USER:$TOKEN "$URL/mobile-node-$VERSION-sources.jar?override=1&publish=1"
+RepoID=$(mvn nexus-staging:rc-list \
+    -settings=bin/package/android/mvn.settings \
+    -DnexusUrl=https://oss.sonatype.org/ \
+    -DserverId=ossrh \
+    | awk '/networkmysterium-/ {id=$2} END{print id}')
+
+mvn nexus-staging:rc-close \
```

Comment:
You should install `mvn` into tooling container `builder`

Reply:
Will it be used anywhere else? Why to install it?

Reply:
How it's related to builder container? It's already too big.

Reply:
So it's already on Travis Machine.. I thought it's not